Test/VimCoreTest/CommonOperationsTest.cs

```diff
@@ -141,7 +141,7 @@ public void DeleteLines_OverFold()
             {
                 Create("cat", "dog", "bear", "fish", "tree");
                 _foldManager.CreateFold(_textView.GetLineRange(1, 2));
-                _operations.DeleteLines(_textBuffer.GetLine(0), 3, VimUtil.MissingRegisterName);
+                _operations.DeleteLines(_textBuffer.GetLine(0), 4, VimUtil.MissingRegisterName);

```

Comment:
I think that `_operations.DeleteLines` should specify the number of unfolded lines.